diff -up cantor-19.11.80/CMakeLists.txt.omv~ cantor-19.11.80/CMakeLists.txt
--- cantor-19.11.80/CMakeLists.txt.omv~	2019-11-18 22:40:59.398449003 +0100
+++ cantor-19.11.80/CMakeLists.txt	2019-11-18 22:41:16.836491128 +0100
@@ -60,7 +60,7 @@ include(KDEFrameworkCompilerSettings)
 include(ECMAddAppIcon)
 include(GenerateExportHeader)
 
-include(thirdparty/CMakeLists.txt)
+add_subdirectory(thirdparty)
 
 if(NOT WIN32)
     set_package_properties(LibSpectre PROPERTIES DESCRIPTION "A PostScript rendering library"
diff -up cantor-19.11.80/thirdparty/CMakeLists.txt.omv~ cantor-19.11.80/thirdparty/CMakeLists.txt
--- cantor-19.11.80/thirdparty/CMakeLists.txt.omv~	2019-11-19 00:28:47.084633440 +0100
+++ cantor-19.11.80/thirdparty/CMakeLists.txt	2019-11-19 00:28:59.485687083 +0100
@@ -29,4 +29,4 @@ add_dependencies(Discount::Lib discount_
 set(Discount_FOUND TRUE)
 
 # preview.sty
-install(FILES thirdparty/standalone.cls DESTINATION  ${KDE_INSTALL_DATADIR}/cantor/latex )
+install(FILES standalone.cls DESTINATION  ${KDE_INSTALL_DATADIR}/cantor/latex )
