--- cantor-16.12.2/src/backends/python/pythonhighlighter.cpp.omv~	2017-02-09 18:16:56.054835003 +0100
+++ cantor-16.12.2/src/backends/python/pythonhighlighter.cpp	2017-02-09 18:18:02.494474822 +0100
@@ -24,6 +24,9 @@
 #include <QTextEdit>
 #include <QDebug>
 
+#define and &&
+#define not !
+
 PythonHighlighter::PythonHighlighter(QObject* parent) : Cantor::DefaultHighlighter(parent)
 {
     qDebug() << "PythonHighlighter construtor";
