--- a/src/backends/R/rserver/CMakeLists.txt.rpath	2010-11-08 13:57:38.000000000 +0100
+++ b/src/backends/R/rserver/CMakeLists.txt	2010-11-08 14:01:35.000000000 +0100
@@ -17,6 +17,8 @@
 add_custom_target(rautoloads COMMAND ${R_EXECUTABLE} --slave --file=${CMAKE_CURRENT_SOURCE_DIR}/tools/autoloads.r > ${CMAKE_CURRENT_BINARY_DIR}/rautoloads.h )
 
 kde4_add_executable( cantor_rserver ${RServer_SRCS} )
+set (CMAKE_SKIP_BUILD_RPATH FALSE)
+set (CMAKE_SKIP_RPATH FALSE)
 target_link_libraries( cantor_rserver ${KDE4_KDEUI_LIBS} cantorlibs ${R_LIBRARIES} ${KDE4_KIO_LIBS})
 add_dependencies(cantor_rserver renvvars rautoloads)
 
